Project.toml

```diff
@@ -3,6 +3,13 @@ uuid = "bd245535-7a0d-4808-be35-e2fe847ca032"
 authors = ["Invenia Technical Computing Corporation"]
 version = "0.1.0"
 
+[deps]
+AxisArrays = "39de3d68-74b9-583c-8d2d-e117c070f3a9"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+NamedDims = "356022a1-0364-5f58-8944-0da4b18d706f"
```

Comment:
need to add `compat` for AxisArrays, Distributions and NamedDims before registering (else won't get auto-merged)